scripts/buildtools/commands/verify-package.js

```diff
@@ -15,8 +15,8 @@ const EXPECTED_BUNDLE_SIZES = {
   'okta-sign-in.no-polyfill.min.js': 1.6 * MB,
   'okta-sign-in.oie.js': 2.4 * MB,
   'okta-sign-in.oie.min.js': 1.2 * MB,
-  'okta-sign-in.polyfill.js': 504 * KB,
-  'okta-sign-in.polyfill.min.js': 108 * KB,
+  'okta-sign-in.polyfill.js': 380 * KB,
+  'okta-sign-in.polyfill.min.js': 90 * KB,
```

Comment:
Yea I am not sure _why_ it decreased in size as we didnt really touch anything in Gen 2, initially I thought vite had something to do with it but we are only using vite for Gen 3 so that can't be it 🤔

Reply:
Gonna dig into this. Smaller sounds better but not if it's because we're not missing polyfills that should be present.

Reply:
1. Did our version of `core-js` change? 
2. Did we change our targets? 
3. Did we change which/how polyfills are included?

All the changes I see are related to path changes and polyfills used.

Big chunk omitted from topic branch side:
https://gist.github.com/lesterchoi-okta/bf1d431ed809decd415daf5f45f26dfa#file-okta-sign-in-polyfill-js-diff-L1-L183 

Example path change:
https://gist.github.com/lesterchoi-okta/bf1d431ed809decd415daf5f45f26dfa#file-okta-sign-in-polyfill-js-diff-L184-L187

Example bundling change:
https://gist.github.com/lesterchoi-okta/bf1d431ed809decd415daf5f45f26dfa#file-okta-sign-in-polyfill-js-diff-L217-L222